keras/trainers/data_adapters/tf_dataset_adapter.py

```diff
@@ -7,7 +7,7 @@
 class TFDatasetAdapter(DataAdapter):
     """Adapter that handles `tf.data.Dataset`."""
 
-    def __init__(self, dataset, class_weight=None):
+    def __init__(self, dataset, class_weight=None, distribution=None):
```

Comment:
Please add a `Args:` section to document the type of each argument (to avoid confusion with tf.distribute)

Reply:
Done.